presidio-anonymizer/presidio_anonymizer/operators/hash.py

```diff
@@ -25,14 +25,14 @@ def operate(self, text: str = None, params: Dict = None) -> str:
         hash_switcher = {
```

Comment:
What are your thoughts about writing a deprecation warning here if the user passes MD5? While it won't break anything (as it would use SHA256 as default), it might not be as the user intends, and the user wouldn't have a way to know.

Reply:
Done